nemo/core/classes/exportable.py

```diff
@@ -172,11 +174,16 @@ def _export(
                         do_constant_folding=do_constant_folding,
                         dynamic_axes=dynamic_axes,
                         opset_version=onnx_opset_version,
+                        export_modules_as_functions=export_modules_as_functions,
                     )
 
                     if check_trace:
                         if isinstance(check_trace, bool):
-                            check_trace_input = [input_example]
```

Comment:
Those are leftovers from debugging - please revert this diff.

Reply:
reverted it back.